### PR TITLE
Add Pdf to Text

### LINKED ISSUE #608: PDF2text
# Aim

- What is the objective of the Script
To convert PDF to Text
# Details

- What the features will your script have
My script converts PDF to Text

### PR DESCRIPTION
**Pull request by muthuannamalai12**
# Description
To convert PDF to Text
Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.

## Fixes #608 (issue_no)

Replace `issue_no` with the issue number which is fixed in this PR

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
- [x] Documentation Update

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

File: PDF2text/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Description: PDF2text
+This is a small script to make a extract text from pdf file.  
+
+### Dependencies:
```
Comment (kaustubhgupta) [inline PDF2text/README.md:4]: level 2(##)

File: PDF2text/script.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+
+pdf_path = input("Enter the path of the pdf file : ")
+
+assert os.path.exists(pdf_path), "this pdf file doesn't exist"
+
+with open(pdf_path, 'rb') as f_r:
+    pdf_pages = pdftotext.PDF(f_r)
+
+for i, page in enumerate(pdf_pages):
```
Comment (kaustubhgupta) [inline PDF2text/script.py:12]: Save the text of the pages in different text files, one text file per page.

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: PDF2text/script.py (hunk 2 above)
Comment (muthuannamalai12) [reply PDF2text/script.py:12]: > Save the text of the pages in different text files, one text file per page.

Can I know how to do it as I am new to it

Comment (kaustubhgupta) [reply PDF2text/script.py:12]: just save the files using:
```python
with open(f"./PDF2text/pages/{i}.txt", "w") as f:
    f.write(page)
```

I think it should work, do make a local run to confirm

**Comment by kaustubhgupta** [comment]
@muthuannamalai12 check the lint code base details for the errors you made

**Comment by kaustubhgupta** [comment]
@muthuannamalai12 did you run the modified script locally? I don't think so

**Comment by muthuannamalai12** [comment]
I cannot able to solve the bug

**Pull request closed without merging**